### DISCUSSION TITLE
How to access REST API secured with self signed certificate?

### DISCUSSION THREAD

**Discussion by tedstriker** [windmill-labs/windmill#3198] [Questions] [answered]
I'm trying to use access a REST API of a self hosted service which encrypts traffic with a self signed certificate.
How do I make workers, especially the Deno part, recognize the CA-certificate where all other self signed certificates are derived from?
I already mounted the certificate by name and its hash as well as the combined `ca-certificates.crt` to `/etc/ssl/certs/`
Also the environment variable `DENO_TLS_CA_STORE=system,mozilla`, `REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt `and `WHITELIST_ENVS=REQUESTS_CA_BUNDLE` have been set as explained here: https://www.lfanew.com/posts/windmill-ca-trust/
For some reason Deno does not use the systems trusted certstore.
How do I make it recognize the custom ca certificate?

**Comment by rubenfiszel** [comment]
What language are you using ? Deno or REST (which is a lightweight Deno runtime)

**Reply by tedstriker** [reply]
I used the REST button in the "Language" section, which opened a TypeScript editor. I don't know how to select DENO directly.

**Reply by rubenfiszel** [reply, marked as answer]
Use: "TypeScript (Deno)"
